examples/todomvc/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+authors = ["Nick Fitzgerald <fitzgen@gmail.com>"]
+edition = "2018"
+name = "dodrio-todomvc"
+version = "0.1.0"
+
+[dependencies]
+cfg-if = "0.1.6"
+futures = "0.1"
+serde_json = "1.0.38"
+wasm-bindgen = "0.2"
+wasm-bindgen-futures = "0.3"
+
+[dependencies.console_error_panic_hook]
+optional = true
+version = "0.1.1"
+
+[dependencies.console_log]
+optional = true
+version = "0.1.2"
+
+[dependencies.dodrio]
+path = "../.."
+
+[dependencies.log]
+optional = true
+version = "0.4.6"
```

Comment:
You might be able to expose some features of this to compile out `debug!` and/or `info!` statements perhaps?

Reply:
This crate itself doesn't do any logging, just `dodrio` but it does need to depend on `log` to get the `Level` when initializing the `console_log` logger. So I think it is OK to tie them together in the `logging` feature since they are only ever used together.

Reply:
Oh for sure, but `debug!`'s code is emitted by default, which means there's a runtime check and the code for formatting is all compiled in. For the tiniest binaries we probably want to compile out the log messages?

Reply:
I've been seeing some `debug!`s get compiled away when there is no logger -- should double check that for `dodrio` for sure though.

Reply:
Oh right that's a good point. I think LTO is good enough to see that if no one sets the max log level the max log level is a constant that forbids logging so everything is optimized away as if it were already compiled out